scheduler/src/cook/mesos/mesos_mock.clj

```diff
@@ -275,8 +295,12 @@
           requested-resources (combine-resources (mapcat :resources tasks))
           resources' (subtract-resources available-resources requested-resources)
           tasks (map #(assoc % :launched-time (t/now)) tasks)
-          task->runtime-ms (-> state :config :task->runtime-ms)]
-      (log/info "Resources requested by tasks: " {:requested-resources requested-resources})
+          task->runtime-ms (-> state :config :task->runtime-ms)
+          new-time-task-id-pairs (map (fn [{:keys [task-id] :as task}]
```

Comment:
Its makes it clear that this structure represent the task completion time. 

Also, when I see a vector I interpret there can be multiple entries for a given task:  [time-1, task-id] and [time-2, task-id].